vm/src/stdlib/os.rs

```diff
@@ -512,33 +577,128 @@ pub fn make_module(vm: &VirtualMachine) -> PyObjectRef {
          "st_size" => ctx.new_property(StatResultRef::st_size),
     });
 
-    py_module!(vm, "_os", {
-        "open" => ctx.new_rustfunc(os_open),
+    let fd_funcs: Vec<(&str, PyObjectRef, Option<bool>, Option<bool>, Option<bool>)> = vec![
+        // name, func_obj, supports_fd, supports_dir_fd, supports_follow_symlinks
+        ("open", ctx.new_rustfunc(os_open), None, Some(false), None),
+        // access Some Some None
+        // chdir Some, None None
+        // chflags Some, None Some
+        // chmod Some Some Some
+        // chown Some Some Some
+        // chroot Some None None
+        (
+            "listdir",
+            ctx.new_rustfunc(os_listdir),
+            Some(false),
+            None,
+            None,
+        ),
+        (
+            "mkdir",
+            ctx.new_rustfunc(os_mkdir),
+            Some(false),
+            Some(false),
+            None,
+        ),
+        // mkfifo Some Some None
+        // mknod Some Some None
+        // pathconf Some None None
+        // readlink Some Some None
+        (
+            "remove",
+            ctx.new_rustfunc(os_remove),
+            Some(false),
+            Some(false),
+            None,
+        ),
+        // rename Some Some None
+        // replace Some Some None
+        (
+            "rmdir",
+            ctx.new_rustfunc(os_rmdir),
+            Some(false),
+            Some(false),
+            None,
+        ),
+        (
+            "scandir",
+            ctx.new_rustfunc(os_scandir),
+            Some(false),
+            None,
+            None,
+        ),
+        (
+            "stat",
+            ctx.new_rustfunc(os_stat),
+            Some(false),
+            Some(false),
+            Some(false),
+        ),
+        (
+            "symlink",
+            ctx.new_rustfunc(os_symlink),
+            None,
+            Some(false),
+            None,
+        ),
+        // truncate Some None None
+        (
+            "unlink",
+            ctx.new_rustfunc(os_remove),
+            Some(false),
+            Some(false),
+            None,
+        ),
+        // utime Some Some Some
+        // walk Some None
```

Comment:
Some of this function like `walk` are implemented in pure python. Maybe consider removing the placeholders?